addons/point_of_sale/static/src/app/notification/notification_service.js

```diff
@@ -0,0 +1,50 @@
+/** @odoo-module */
+
+import { reactive, Component } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { Transition } from "@web/core/transition";
+
+const TRANSITION_LEAVE_DURATION = 200; // ms
+
+class Notification extends Component {
+    static template = "point_of_sale.Notification";
+    static props = { message: String, close: Function, className: String };
+}
+
+class NotificationContainer extends Component {
+    static template = "point_of_sale.NotificationContainer";
+    static components = { Transition, Notification };
+    static props = {
+        notifications: Object,
+    };
+    leaveDuration = TRANSITION_LEAVE_DURATION;
+}
+// FIXME: probably should use the main notification service from web long term.
+export const notificationService = {
```

Comment:
Shouldn't this be split in another file? One handling the component(s) and one handling the service?

Reply:
Since I want to remove this completely in the end I don't think it matters too much, especially since the components are so small